src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -1461,7 +1461,7 @@ namespace EvaporativeFluidCoolers {
                 }
             }
             if (state.dataPlnt->PlantFirstSizesOkayToFinalize) {
-                if (this->TypeOf_Num == DataPlant::TypeOf_EvapFluidCooler_SingleSpd) {
+                if (this->TypeOf_enum == DataPlant::PlantEquipmentType::EvapFluidCooler_SingleSpd) {
                     if (state.dataPlnt->PlantFinalSizesOkayToReport) {
                         BaseSizer::reportSizerOutput(
                             state, cEvapFluidCooler_SingleSpeed, this->Name, "Fan Power at Design Air Flow Rate [W]", this->HighSpeedFanPower);
```

Comment:
Should be a lookup in `constexpr std::array<std::string_view>` and probably inside the reporting function.

Reply:
Did you mean the string "Fan Power at Design Air Flow Rate [W]"?